### PR TITLE
FIX: fixes for ios devices

### PR DESCRIPTION
**Pull request by rastajpa**

### CODE REVIEW

File: src/components/action-sheet/action-sheet.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <div class="backdrop" [ngClass]="{'backdrop-fade-in': slideIn}" (click)="dismiss()"></div>
-<div class="action-sheet" [ngClass]="{'opened': slideIn, 'closed-top': !slideIn && fromTop, 'closed-bottom': !slideIn && !fromTop, 'action-sheet-top': fromTop, 'action-sheet-bottom': !fromTop }">
+<div class="action-sheet" [ngClass]="{'opened': slideIn, 'closed-top': !slideIn && fromTop, 'closed-bottom': !slideIn && !fromTop, 'action-sheet-top': fromTop, 'action-sheet-bottom': !fromTop, 'safe-top-area': fromTop }">
```
Comment (cmgustavo) [inline src/components/action-sheet/action-sheet.html]: it's too complex to read. Why not group class with same condition?

File: src/components/action-sheet/action-sheet.scss [hunk 2]
```diff
@@ -61,4 +61,8 @@ action-sheet {
       transform: translateY(calc(-100% + 8px));
     }
   }
+  .safe-top-area {
+    padding-top: calc(10px + env(safe-area-inset-top));
```
Comment (cmgustavo) [inline src/components/action-sheet/action-sheet.scss]: 10px is applied to desktop too. Is that the idea?

File: src/components/action-sheet/action-sheet.html (hunk 1 above)
Comment (rastajpa) [reply src/components/action-sheet/action-sheet.html]: you're right!

File: src/components/action-sheet/action-sheet.scss (hunk 2 above)
Comment (rastajpa) [reply src/components/action-sheet/action-sheet.scss]: it is on purpose. It looks more similar to the design

**Pull request merged**